client/src/components/License/License.vue

```diff
@@ -6,9 +6,7 @@
             {{ title }}
         </span>
         {{ license.name }}
-        <a target="_blank" :href="license.url">
-            <font-awesome-icon icon="external-link-alt" />
-        </a>
+        <external-link :href="license.url" />
```

Comment:
```suggestion
        <external-link :href="license.url">  {{ license.name }} </external-link>
```

Reply:
I don't think this is going to work in the other contexts this component is used in.  We can flag it for followup, though -- I can do that in the same pass I convert over the other external-link bits.